cluster/gce/configure-vm.sh

```diff
@@ -386,7 +378,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: '$(echo "$MINION_IP_RANGE" | sed -e "s/'/''/g")'
+  cbr-cidr: 10.123.45.0/30
```

Comment:
Here's I imagine it would work: A pod created via manifest will have a black-holed IP address until kubelet flips crb0 and then kicks docker. Docker will kill the container, and when it restarts kubelet should restart the pod, at which point it should get a routable IP from docker. 

Does that sound correct?